### PR TITLE
Fixed code

### PR DESCRIPTION
**Pull request by MatthewZMSU**
Improved realization of recaptcha_solver, changed comments

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -23,8 +27,36 @@ const HEADLESS = (process.env.HEADLESS || "true").toLowerCase() === "true";
 const CONNECT_TIMEOUT = parseInt(process.env.CONNECT_TIMEOUT) || 180000;
 const VIEWPORT_WIDTH = parseInt(process.env.VIEWPORT_WIDTH) || 1280;
 const VIEWPORT_HEIGHT = parseInt(process.env.VIEWPORT_HEIGHT) || 720;
+const TWO_CAPTCHA_TOKEN = process.env.TWO_CAPTCHA_TOKEN || "0";
```
Comment (Yatskov) [inline app.js:30]: Maybe we should name this 2CAPTCHA_TOKEN for easier mapping

File: routes/recaptcha_solver.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+ */
+
+async function action(page, request) {
+    const {
+        captchas,
+        filtered,
+        solutions,
+        solved,
+        error
+    } = await page.solveRecaptchas();
```
Comment (Yatskov) [inline routes/recaptcha_solver.js:28]: We want such feedback as a first action in formResponse. Also we should have possibility to turn off captcha solving in request params.

**Review by Yatskov — CHANGES_REQUESTED** [review]

**Comment by Yatskov** [comment]
Also i ask to rename pull request title

File: app.js (hunk 1 above)
Comment (MatthewZMSU) [reply app.js:30]: Variables cannot start with digit literal!

**Pull request closed without merging**